app/controllers/application_controller/compare.rb

```diff
@@ -261,13 +261,24 @@ def compare_to_pdf
   end
 
   # Create an MIQ_Report object from a compare object
-  def create_compare_report(csv = false)
+  def create_compare_or_drift_report(compare_or_drift, csv = false)
```

Comment:
Also, this method should probably be a private method, since it's being called by the other two methods internally.